### DISCUSSION TITLE
No entiendo como funciona el link para probar la compilación en otro pc

### DISCUSSION THREAD

**Discussion by TomasAuad** [IIC2133-PUC/2024-1#118] [Ayuda] [answered]
Hola estoy intentando acceder a https://edd.ing.puc.cl/run/T1-2024-1/TomasAuad para probar mi codigo la pagina no encuentra mi usuario: User not found. 

Hay alguna manera especial de correrlo?
Muchas gracias.

**Comment by benjavicente** [comment, marked as answer]
Arreglado
